website/src/content/api-reference/commercetools-frontend-application-shell-connectors.mdx

```diff
@@ -0,0 +1,319 @@
+---
+title: "@/application-shell-connectors"
+---
+
+<Subtitle>
+
+Components to work with the `@commercetools-frontend/application-shell` package to access and subscribe to specific data within the AppShell.
+
+</Subtitle>
+
+# Installation
+
+```
+yarn add @commercetools-frontend/application-shell-connectors
+
+# or
+
+npm --save install @commercetools-frontend/application-shell-connectors
+```
+
+Additionally install the peer dependencies (if not present)
+
+```
+yarn add react @apollo/client
+
+# or
+
+npm --save install react @apollo/client
+```
+
+# Context
+
+In this section we describe the type of information you will find in the main application context and how to access it.
+
+## ApplicationContext
+
+This is the main React context for the application and it holds global information regarding user, permissions, project, etc.
+
+To read more about the data in the context, read the following;
```

Comment:
Missing link to types maybe? In parts we should maybe note that it's two parts. One is what we provide and a user can't influence. That's user, permissions, project. Then there is what a user can put onto the context which usually relates to `environment`.